src/engines/L3/fontLoader.js

```diff
@@ -16,10 +16,24 @@ export default () => {
         font.json = font.file.replace(/\.[^.]+$/, `.${font.type}.json`)
       }
       stage.fontManager.addFontFace(
-        new SdfTrFontFace(font.family, {}, font.type, stage, font.png, font.json)
+        new SdfTrFontFace(font.type, {
+          fontFamily: font.family,
+          descriptors: {},
+          atlasUrl: font.png,
+          atlasDataUrl: font.json,
+          stage,
+          ...(font.metrics ? { metrics: font.metrics } : {}),
```

Comment:
can't we just do:

```
metrics: font.metrics || {}
````

Reply:
I tried this @michielvandergeest  Getting problem at this line of code in renderer
https://github.com/lightning-js/renderer/blob/main/src/core/text-rendering/font-face-types/TrFontFace.ts#L150

Reply:
can I change it to `metrics: font.metrics || null `

Reply:
@suresh-gangumalla Actually, it can be 

```js
metrics: font.metrics
```

 directly because the renderer code checks it with an if statement (as you pointed out), so if `font.metrics` does not exist (`undefined`) it will processed as the same with `null` in the renderer code.

Reply:
Got it @uguraslan 
You are right. we can't just pass an empty {} but instead `undefined ` is fine